### ISSUE TITLE
[Docs]: Add a layout file example

### ISSUE THREAD

**Issue by JoltCode** [remix-run/react-router#12507]
### Describe what's incorrect/missing in the documentation

Got a little confused whether to use `<Outlet />` or whether the children were meant to be automatically given through to the props for the component in my layout file, so would be good to add this as an example to the [layout route section](https://github.com/remix-run/react-router/blob/main/docs/start/framework/routing.md#layout-routes)!

Basically, should just duplicate the part under the [Nested Routes section](https://github.com/remix-run/react-router/blob/main/docs/start/framework/routing.md#nested-routes), or state that it works the same way.

**Comment by brookslybrand** [comment]
Yep, this was confusing

Fortunately someone in the community tackled it in #12548!

Sorry this issue remained open a bit longer than necessary
